packages/fether-electron/src/main/app/methods/showTrayBalloon.js

```diff
@@ -18,15 +18,17 @@ const iconBalloonPath = path.join(
 
 const pino = Pino();
 
+// Supported only on Windows OS
 function showTrayBalloon (fetherApp) {
   const { tray } = fetherApp;
 
   pino.info('Showing Tray Balloon');
 
   tray.displayBalloon({
-    content: `Press ALT in the Fether window to toggle the menu`,
+    content:
+      'Click to toggle Fether window. Right-click Fether window to toggle Fether menu',
```

Comment:
```suggestion
      'Show/Hide Fether',
```

This is not true for Linux. Since it's a menu item we should keep things short and simple.

Reply:
How should we communicate to users that they can view the full menu by right-clicking on the Fether window?